docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
+      value = aiven_cassandra.example_cassandra.service_port
+    }
+
+    output "cassandra_service_username" {
+    value = aiven_cassandra.example_cassandra.service_username
+    }
+
+    output "cassandra_service_password" {
+      value     = aiven_cassandra.example_cassandra.service_password
+      sensitive = true
+    }
+    ```
+
+1. Run `terraform plan` > `terraform apply`.
+
+</TabItem>
+<TabItem value="3" label="K8s">
+1. Update file `cassandra-sample.yaml`:
+
+   - Add `service_log: true` and `terminationProtection: true`.
+   - Update `maintenanceWindowDow: sunday` and `maintenanceWindowTime: 22:00:00`.
+
+   ```yaml
+   apiVersion: aiven.io/v1alpha1
+   kind: Cassandra
+   metadata:
+     name: cassandra-sample
+   spec:
+     authSecretRef:
+       name: aiven-token
+       key: token
+
+     connInfoSecretTarget:
+       name: cassandra-secret
+
+     userConfig:
+       migrate_sstableloader: true
+       service_log: true
+
+     project: PROJECT_NAME
+     cloudName: google-europe-west1
+     plan: startup-4
+
+     maintenanceWindowDow: sunday
+     maintenanceWindowTime: 22:00:00
+     terminationProtection: true
+   ```
+
+1. Update the service by applying the configuration:
+
+   ```shell
+   kubectl apply -f cassandra-sample.yaml
+   ```
+
+1. Review the resource you updated with the following command:
+
+   ```shell
+   kubectl describe cassandra.aiven.io cassandra-sample
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
+
+See the available configuration options in
+[Aiven Operator for Kubernetes®: Cassandra](https://aiven.github.io/aiven-operator/api-reference/cassandra.html).
+</TabItem>
+</Tabs>
+
+## Connect to the service{#connect-to-service}
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Console" default>
+1. Log in to the [Aiven Console](https://console.aiven.io/), and go to your
+   organization > project > Aiven for Apache Cassandra service.
+1. On the <ConsoleLabel name="overview"/> page of your service, click
+   **Quick connect**.
+1. In the **Connect** window, select a tool or language to connect to your service, follow
+   the connection instructions, and click **Done**.
+
+   ```bash
+   # Sample command for connecting with the cqlsh client
+
+   ./cqlsh --ssl -u USER_NAME -p PASSWORD HOST_NAME PORT_NUMBER
+   ```
+
+</TabItem>
+<TabItem value="2" label="Terraform">
+Access your new service with the Apache Cassandra client using the environment variables
+assigned to Terraform outputs:
 
 ```bash
-avn service create demo-cassandra       \
-   --service-type cassandra             \
-   --cloud CLOUD_AND_REGION             \
-   --plan PLAN                          \
-   --project PROJECT_NAME
+./cqlsh --ssl --host=$CASSANDRA_HOST --port=$CASSANDRA_PORT --user=$CASSANDRA_USER --password=$CASSANDRA_PASSWORD
 ```
 
-:::note
+</TabItem>
+<TabItem value="3" label="cqlsh client">
```

Comment:
Since all the tabs on the page are in the same group (group1) this tab with value 3, syncs with tabs with value 3 above, but those tabs are about K8s, so this tab needs a new value to be independant.

```suggestion
<TabItem value="cqlsh" label="cqlsh client">
```